xline/src/client/convert.rs

```diff
@@ -96,164 +86,3 @@ impl From<&LeaseTimeToLiveRequest> for LeaseTimeToLiveOptions {
         opts
     }
 }
-
-impl From<EtcdPutResponse> for PutResponse {
```

Comment:
Why you just move resposnes to new crate?

Reply:
Because the request are wrapper types defined in `xline/src/client`, so they can be implement there. And the resposne types are rpc types in the proto files and can only be implemented in `xlineapi`.